### DISCUSSION TITLE
[app/docs/ssl.md] Enable SSL - using python3

### DISCUSSION THREAD

**Discussion by 43skyline** [simple-login/app#717] [Q&A] [answered]
While following the script, 

> sudo apt-get install certbot python-certbot-nginx

Under Ubuntu 20.04, this is returning

> Package python-certbot-nginx is not available, but is referred to by another package.
> This may mean that the package is missing, has been obsoleted, or
> is only available from another source
> However the following packages replace it:
>   python3-certbot-nginx

It appears python-certbot-nginx is replaced with python3-certbot-ngix.

Is it all good to go with python3-certbot-ngix?

**Comment by nguyenkims** [comment, marked as answer]
Yes I think it's safe to go with `python3-certbot-nginx`
